vllm/core/block_manager_v2.py

```diff
@@ -194,17 +193,23 @@ def get_block_table(self, seq: Sequence) -> List[int]:
         assert all(b is not None for b in block_ids)
         return block_ids
 
-    def access_all_blocks_in_seq(self, seq, now):
-        # TODO add prefix caching support.
-        # Tracked here https://github.com/vllm-project/vllm/issues/3667
-        pass
+    def access_all_blocks_in_seq(self, seq: Sequence, now: float):
+        if self.enable_caching:
```

Comment:
nit: add comment why we only do this for prefix caching (seems performance?)

Reply:
Got it, would apply~